### PR TITLE
bears/bootstrap: Add BootLintBear

### PR DESCRIPTION
**Pull request by mr-karan**
Integrated Bootstrap Linter Bear

Fixes https://github.com/coala-analyzer/coala-bears/issues/143

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
   "dependencies": {
     "alex": "~2.0.1",
+    "bootlint": "~0.14.2"
```
Comment (Redridge) [inline package.json]: There is a `,` missing at the end of this line

**Comment by sils1297** [comment]
hey your tests fail, can you fix that?

**Comment by sils1297** [comment]
apparently missing a doctype in some test document?

**Comment by sils1297** [comment]
or rather your bear doesn't handle the missing doctype thing well it seems

**Comment by AbdealiJK** [comment]
ack  e86beee

**Comment by AbdealiJK** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by AbdealiJK** [comment]
@rultor merge

**Pull request merged**